package.json

```diff
@@ -43,12 +43,13 @@
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --ext .js,.jsx .",
     "lint:scss": "stylelint 'src/**/*.scss' --syntax scss",
-    "start": "tt-serve",
+    "start": "NODE_OPTIONS=--max-old-space-size=8192 tt-serve",
```

Comment:
This makes me sad.

Reply:
I think we're right on the edge and could remove this if we get some duplicates cleaned up.... see my other PRs